Observer/TescoPaymentObserver.php

```diff
@@ -0,0 +1,99 @@
+<?php
+namespace Omise\Payment\Observer;
+
+use Magento\Framework\Event\ObserverInterface;
+use Magento\Payment\Model\InfoInterface;
+use Magento\Framework\Event\Observer;
+
+class TescoPaymentObserver implements ObserverInterface
+{
+    /**
+    * @var \Magento\Checkout\Model\Session
+    */
+    private $_checkoutSession;
+
+    /**
+    * @var \Magento\Framework\App\Config\ScopeConfigInterface
+    */
+    private $_scopeConfig;    
+    
+    /**
+    * @var  \Magento\Framework\Mail\Template\TransportBuilder
+    */
+    private $_transportBuilder;
+    
+    /**
+    * @var  \Omise\Payment\Helper
+    */
+    private $_helper;
+
+    public function __construct(
+        \Omise\Payment\Helper\OmiseHelper $helper,
+        \Magento\Checkout\Model\Session $checkoutSession,
+        \Magento\Framework\Mail\Template\TransportBuilder $transportBuilder,
+        \Magento\Framework\App\Config\ScopeConfigInterface $scopeConfig
+    ) {
+        $this->_scopeConfig = $scopeConfig;
+        $this->_helper = $helper;
+        $this->_checkoutSession = $checkoutSession;
+        $this->_transportBuilder = $transportBuilder;
+    }
+
+
+    /**
+     * Set forced canCreditmemo flag
+     *
+     * @param \Magento\Framework\Event\Observer $observer
+     * @return $this
+     */
+    public function execute(Observer $observer)
+    {
+        $order = $this->_checkoutSession->getLastRealOrder();
+        $paymentData = $order->getPayment()->getData();
+
+        if ($paymentData['additional_information']['payment_type'] !== 'bill_payment_tesco_lotus') {
+            return $this;
+        }
+
+        $storeName =  $this->_scopeConfig->getValue(
```

Comment:
@jacstn code styling, please remove 1 space after `=` at `$storeName =  $this->...`